### ISSUE TITLE
OBO importer variable not substituted in message

### ISSUE THREAD

**Issue by dsenalik** [tripal/tripal#1904]
### Tripal Version

4.x

### Drupal Version

not applicable

### PostgreSQL Version

not applicable

### PHP Version

not applicable

### General Description

This block of code in the OBO importer
```
    $message = t('Found the following namespaces: @namespaces.',
      ['@namespaces' => implode(', ', array_keys($this->obo_namespaces))]);
    foreach ($this->obo_namespaces as $namespace => $cv) {
      $this->insertChadoCv($namespace);
    }
    $this->logger->notice($message->getUntranslatedString());
```
always prints 
`Found the following namespaces: @namespaces.`
because of that `getUntranslatedString()` in there.

### Steps to reproduce

To reproduce, run prepare chado, or even easier build a docker.

### Error messages

```shell
.
```

### Screenshots

.

**Comment by pechtelt1** [comment]
Hi @dsenalik! I'm new to contributing to open source, but want to get started with it. I'm bit familiar with PHP and trying to learn better with contributing to open-source projects. I only don't know what you want to be the solution. 

This will be fixed when removing the ->getUntranslatedString() right?

**Comment by dsenalik** [comment]
Thanks for wanting to contribute @pechtelt1 - Sorry but I already fixed this with #1905.
If you want a different issue to work on you could try #1894
I haven't looked yet at exactly where the change should be, but I can add that to #1894 later...

**Comment by pechtelt1** [comment]
Thanks @dsenalik for your reply! Yesterday evening i had some time. Until saturday pretty busy. If it is still open until then i would like work on it! :)
